lib/lumberjack/client.rb

```diff
@@ -114,18 +109,28 @@ def write(msg)
     end
 
     public
-    def write_hash(hash)
-      frame = Encoder.to_compressed_frame(hash, inc)
-      ack if unacked_sequence_size >= @window_size
-      write frame
+    def write_hash(elements)
```

Comment:
Also, if we're moving to ack here, we should make the method name something more appropriate I think

write_sync? write_and_wait?